apps/tng-radio-hub/src/Components/widgets/navbar/navbar.tsx

```diff
@@ -0,0 +1,20 @@
+import { Link } from 'react-router-dom';
+import './navbar.module.css';
+
+/* eslint-disable-next-line */
+export interface NavbarProps {}
```

Comment:
Remove unused props. Better yet, you might want to actually use the props to feed in the data for the links. That would make this component "open to extension but closed to modifications".